src/azure-cli/azure/cli/command_modules/acs/_consts.py

```diff
@@ -12,6 +12,13 @@
 CONST_SPOT_EVICTION_POLICY_DELETE = "Delete"
 CONST_SPOT_EVICTION_POLICY_DEALLOCATE = "Deallocate"
 
+# consts for upgrade channel
```

Comment:
Found this comment on an old PR. Not sure if we should keep the same style as the old implementation for preview, or try to follow this comment:
https://github.com/Azure/azure-cli-extensions/pull/2694#discussion_r526526008

Reply:
It is recommended to follow this comment and define them as enumeration in the SDK if possible

Reply:
I might have missed how this would be done, but I did not see another example of this, so I don't plan on breaking the current practice.